jreleaser.yml

```diff
@@ -74,23 +74,43 @@ release:
 
 distributions:
   wave-cli:
-    type: NATIVE_IMAGE
+    type: FLAT_BINARY
     artifacts:
       - path: "nativeCompile-ubuntu-latest/wave"
         transform: "wave-{{projectEffectiveVersion}}-linux-x86_64"
         platform: linux-x86_64
-      - path: "nativeCompile-macos-latest/wave"
-        transform: "wave-{{projectEffectiveVersion}}-macos-x86_64"
-        platform: osx-x86_64
-      - path: "nativeCompile-macos-latest-xlarge/wave"
-        transform: "wave-{{projectEffectiveVersion}}-macos-arm64"
-        platform: osx-arm64
+        extraProperties:
+          graalVMNativeImage: true
       - path: "nativeCompile-windows-latest/wave.exe"
         transform: "wave-{{projectEffectiveVersion}}-windows-x86_64.exe"
         platform: windows-x86_64
+        extraProperties:
+          graalVMNativeImage: true
+      - path: "nativeCompile-macos-latest/wave.zip"
+        transform: "wave-{{projectEffectiveVersion}}-macos-x86_64.zip"
+        platform: osx-x86_64
+        extraProperties:
+          graalVMNativeImage: true
+      - path: "nativeCompile-macos-latest-xlarge/wave.zip"
+        transform: "wave-{{projectEffectiveVersion}}-macos-arm64.zip"
+        platform: osx-aarch_64
+        extraProperties:
+          graalVMNativeImage: true
 
   wave-cli-jar:
     type: SINGLE_JAR
     artifacts:
       - path: "wave_jar/wave.jar"
-        transform: "wave-{{projectEffectiveVersion}}.jar"
\ No newline at end of file
+        transform: "wave-{{projectEffectiveVersion}}.jar"
+
+packagers:
+  brew:
+    active: ALWAYS
+    continueOnError: false
+    templateDirectory: ./app/src/jreleaser/distributions/wave/brew
```

Comment:
Should this path, line 110, also change one of the dir names from `wave` to `wave-cli`? 
And the dir path of the formula template as well?
or maybe I am missing/confusing something?

Reply:
There are Seqera Labs in the License, but I think we can change it in another PR
I will change this in Jreleaser in this PR

Reply:
ok i will change it to wave-cli